internal/services/users/user_data_source.go

```diff
@@ -359,6 +377,7 @@ func userDataSourceRead(ctx context.Context, d *schema.ResourceData, meta interf
 	tf.Set(d, "department", user.Department)
 	tf.Set(d, "display_name", user.DisplayName)
 	tf.Set(d, "employee_id", user.EmployeeId)
+	tf.Set(d, "employee_type", user.EmployeeType)
```

Comment:
It looks like `cost_center` and `division` are missing here?

Reply:
good point; added at the bottom of the function